### ISSUE TITLE
[DX] Allow elements to be visually hidden when using the `visually-hidden` class (in addition to `element-invisible`)

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5767]
We are using the `element-invisible` class in CSS when we want to visually hide elements in an A11Y-friendly way (see https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/css/system.css#L280), but Drupal and Bootstrap are using `visually-hidden` instead. See:
- https://getbootstrap.com/docs/5.0/helpers/visually-hidden
- https://www.drupal.org/docs/accessibility/hide-content-properly

To make it easier for people to use code snippets from popular sources with as fewer changes required, I think that we should be allowing developers to use either class, and have it produce the same effect.

**Comment by indigoxela** [comment]
Good point, many times I had to take a look, what the right class is for the CMS I'm currently working with.

One thing to consider: if we start to provide both class names (sort of synonyms), we probably have to keep them both for a while, probably until 2.x, possibly forever. But would it do any harm?

**Comment by klonos** [comment]
> But would it do any harm?

I think not. There are currently only 4 instances of this class being used in our CSS, with only 3 of them being actual CSS - the other one is used in an inline comment/docblock. All of these instances are in the same section of the [system.css](https://github.com/backdrop/backdrop/blob/1.x/core/modules/system/css/system.css#L280) file, and they rarely ever change.
